shared/chat/conversation/notices/create-team-notice/container.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as Constants from '../../../../constants/chat'
+import CreateTeamNotice from '.'
+import {compose} from 'recompose'
+import {connect} from 'react-redux'
+import {navigateAppend} from '../../../../actions/route-tree'
+
+import type {TypedState} from '../../../../constants/reducer'
+import type {StateProps, DispatchProps} from './container'
+
+const mapStateToProps = (state: TypedState) => {
+  const selectedConversationIDKey = Constants.getSelectedConversation(state)
+  if (!selectedConversationIDKey) {
```

Comment:
i'm hesitant about this kind of error. We had this in attachments and there was race issues where this would get triggered. we can leave it but don't forget mapStateToProps gets called immediately on any store change (like on a logout)